packages/@yodaos/speech-synthesis/index.js

```diff
@@ -18,6 +18,9 @@ var Status = {
 
 var Events = ['start', 'end', 'cancel']
 
+var endoscope = require('@yoda/endoscope')
+var SynthesizerEventMetric = new endoscope.Enum('yodaos:speech-synthesis:event', { labels: [ 'id', 'text' ], states: Events.concat('error') })
```

Comment:
This line is oversized :)

Reply:
As for the variable name, the instance should keep the first char be in lower-case.